### ISSUE TITLE
NFT Marketplace MVP

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/atlas#1873]
**Context**

Epic encapsulating addition of new sections to enhance the NFT assets discovery and navigation

## Sections

- [ ] We need to do competitive reasearch see how NFTs are broadly categorised on home/ landing pages, source that back to the team, discuss and agree the sections we are launching with subject to tech feasibility.

⚠️ the list needs to be revised:
* [ ] Newly Minted (newly minted last 30 days, sorted by asc date of minted)
* [ ] HOT (minted over last 3 mons, with highest bids / by now prices)
* [ ] Recently sold
* [ ] Whitelisted

☝️ Out of scope:

* [ ] #2904
* [ ] #2905

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201958687417145/1202496765567223) by [Unito](https://www.unito.io)

**Comment by KubaMikolajczyk** [comment]
Competitive research & Explorations are done ✅
🎥 Loom video: https://loom.com/share/aea2de75c46c43b18ec492608bf52d20
📄 Figma research: https://www.figma.com/file/T91xLMaDc5tjGDWIVldv2h/NFT-Marketplace?node-id=803%3A174
📄 Figma explorations: https://www.figma.com/file/T91xLMaDc5tjGDWIVldv2h/NFT-Marketplace?node-id=1212%3A41768

**Comment by dmtrjsg** [comment]
@KubaMikolajczyk solid work 💪 My initial suggestion was to add a few sections, but I really like where you are taking it! 

1. Decorating the landing screen is good, How about a hero banner with the Section from Joystream.org on the very top with the CTAs "Explore All NFTs/ Mint"?
2. Let's keep the search bar on top for the consistent header across all viewers experience
3. I think 'Trending' is a common practice for all content products. Doing this on page views is unreliable on the current infra, so suggest we go by number of bids placed and only show open/ timed auctions, and exclude buy now from this section.
4. 'Newly minted' section seems quite straightforward and could accommodate all types of auctions including buy now sale.
5. 'Recently Purchased' seems also to be within the realms of feasibility, and if we can  take last 15 and sort it by date settled
6. Each section would be 3 rows preview and lead to a dedicated page of 100 in a grid view with Load more/ paginated view (tbc)
7. 'Featured' would be good to add on the very top, and optional, based on some sort of whitelist set up by relevant WG, but we probably need to drive it by Martin to validate if that's within the range of their appetite and validate the operations behind it.
8. 'Recommended creators' - same as featured, would be great to have it but also optional
9. 'Top Sellers' - would be great to feature these in a separate section / carousel, but I understand this is outside of QN feasibility RN, tbc on the call tomorrow
10. 'Auctions ending soon' within next 3 days, could be another section featuring only Timed Auctions ordered by time to ending desc
12. Explainer Video is a good idea, but I would serve it in the overlay modal on the first visit of the page and "don't show this again" Stored in local storage

To Summarise, I think it's a good idea to have some sort of a landing page showing different sections, and a dedicated page per section with the filters that we already have in each dedicated section page.

**Comment by bedeho** [comment]
# Note on competitive research

I think we should be using competitor research, in general, as a way to find inspiration and references on addressing known problems we have, either indicated by data or some other form of qualitative user feedback, **at least when we are not in the 0-to-1 stage of some product feature**, but rather doing refinements or improvements. Other products will have different scale, user bases, operator models and business models, and it will be very difficult to just review and import without more constraints.

# NFT Marketplace MVP timing

While I see a lot of attractive visuals here, and I really appreciate the work you did reviewing incumbent marketplaces, I think the problem statement here was too open ended. We are not starting with any particular known user problem in mind, and I would argue that it's probably premature to explore this without that. This feature set in particular too early to focus on because we don't know what the pace or scale of adoption for our NFT features will be - and the product features (e.g. search), would be heavily influenced by this, and this pace will be heavily impacted by exogenous factors like learning creator outreach, fiat-integration timing and general YPP rollout.

# Feedback

Here are some general points:

1. I think the ideal of 
2. We have to be very careful about featuring based on pure user metrics, like views or likes, as this is highly gameable, and there are incentives to game this in our case, not only for abuse but also for gain. I don't know how other marketplaces manage this, probably with reasonably sophisticated abuse detection and manual intervention, but it may not be feasible for us to rely on that at this early stage. For this reason, I highly suggest we rely on featuring controlled by the application operator over purely automated discovery.
3. I can't really think of a use case for pure search over NFTs as a way to discover. This makes the most sense for me for users on very large marketplaces who are just trying to find "something" to buy and invest in, which really feels like it caters more to pure speculation, and this is a bit different to my mental model of who will be buying our NFTs. Open to hearing other points of view.
4. I find it very difficult to judge whether one page or two separate pages is best. 
5. I know its a hassle, but I think having more hetrogenous and ecological preview images & titles will give a better feeling about how different design alternatives really feel.

**Comment by dmtrjsg** [comment]
Agreed to park this topic till we get the problem better refined: 
- more insight into the marketing channels of acquisition that work
- appetite of users to purchase NFTs from unfamiliar creators vs being brought to the app by specific creators and purchasing from their channel page
- conduct research interviews with creators to get their view and expectations
- understand the content curation feasibility both technically and operationally

In the meantime, the suggestion was to include NFTs support to search requests, previews and results. @toiletgranny will drive this work upon coming back from holidays.

cc @bedeho

**Comment by dmtrjsg** [comment]
Correction:
- parking all NFT related issues to be properly user-feedback driven. 
cc @toiletgranny

**Comment by dmtrjsg** [comment]
Implementation Epic:
- https://github.com/Joystream/atlas/issues/3686
